src/libraries/Microsoft.Extensions.Diagnostics.Abstractions/ref/Microsoft.Extensions.Diagnostics.Abstractions.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.Metrics
```

Comment:
`Microsoft.Extensions.Diagnostics.Abstractions` assembly only contains the `Microsoft.Extensions.Metrics` namespace. Is there prior art of `Microsoft.Extensions.*` packages containing a different namespace than its assembly name?

I wonder whether the namespace should be `Microsoft.Extensions.Diagnostics.Metrics`. That aligns with `System.Diagnostics.Metrics`.